### DISCUSSION TITLE
ScriptReference/SceneManagement.Scene.GetRootGameObjects

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#587] [Notes]
# ScriptReference/SceneManagement.Scene.GetRootGameObjects

https://docs.unity3d.com/ScriptReference/SceneManagement.Scene.GetRootGameObjects.html

**Comment by electronics45** [comment]
Regarding the overload that takes a list:
- The method will call list.clear() internally.  You don't need to do this manually.
- Despite the documentation, you do NOT need to ensure the capacity of the list.  It is checked and increased internally before being filled.
